pkg/config/config.go

```diff
@@ -51,6 +52,18 @@ type GrpcConfig struct {
 	MaxMessageSizeBytes int  `json:"maxMessageSizeBytes" pflag:",The max size in bytes for incoming gRPC messages"`
 }
 
+// KubeClientConfig contains the configuration used by flytepropeller to configure its internal Kubernetes Client.
+type KubeClientConfig struct {
+	// QPS indicates the maximum QPS to the master from this client.
+	// If it's zero, the created RESTClient will use DefaultQPS: 5
+	QPS int32 `json:"qps" pflag:",Max QPS to the master for requests to KubeAPI. 0 defaults to 5."`
+	// Maximum burst for throttle.
+	// If it's zero, the created RESTClient will use DefaultBurst: 10.
+	Burst int `json:"burst" pflag:",Max burst rate for throttle. 0 defaults to 10"`
+	// The maximum length of time to wait before giving up on a server request. A value of zero means no timeout.
+	Timeout config.Duration `json:"timeout" pflag:",Max duration allowed for every request to KubeAPI before giving up. 0 implies no timeout."`
+}
```

Comment:
I feel the way this is used is not encompassing the entire kubeClientConfig but just the rest API configuration.
Basically we have the connection parameters coming from one place
* kubeConfigPath 
* runtimeInterfaces.ClusterConfig
But just the rest Api kube config from this . 
Can we make this specific to mean that so that its not confusing

Reply:
Do you suggest just renaming this configuration?

So I copied the config definition straight from propeller. IMO keeping these things uniform between the repos makes sense, but we already changed that by using camel instead of kebob case. Happy to change this however you suggest!

Reply:
I am ok with the current one with kebob case.